### ISSUE TITLE
Array out of bounds failure

### ISSUE THREAD

**Issue by mattdturner** [CICE-Consortium/Icepack#45]
When running the test suite on Onyx (a Cray XC40/50 using Intel), the model fails to run to completion for case onyx_smoke_col_1x1_alt01_debug_run1year.  The relevant portion of the runlog file is:
~~~~
 WARNING - Zero-layer thermodynamics
  Initial forcing data year =         2015
  Final   forcing data year =         2015
forrtl: severe (408): fort: (2): Subscript #1 of the array ISWABS has value 2 which is 
greater than the upper bound of 1

Image              PC                Routine            Line        Source
icepack            000000002042E836  Unknown               Unknown  Unknown
icepack            0000000020229DC0  icepack_shortwave         653  icepack_shortwave.F90
icepack            0000000020227E2D  icepack_shortwave         257  icepack_shortwave.F90
icepack            00000000202A8662  icepack_shortwave        4134  icepack_shortwave.F90
icepack            0000000020086929  icedrv_init_colum         189  icedrv_init_column.F90
icepack            00000000200013F3  icedrv_initmod_mp         108  icedrv_InitMod.F90
icepack            0000000020002037  MAIN__                     40  icedrv_MAIN.F90
icepack            000000002000120E  Unknown               Unknown  Unknown
icepack            00000000204F27F0  Unknown               Unknown  Unknown
icepack            00000000200010F7  Unknown               Unknown  Unknown
~~~~

**Comment by apcraig** [comment]
This is happening on all machines and needs to be fixed.  Thanks for creating an issue.

**Comment by eclare108213** [comment]
This issue is fixed in Icepack PR #50
